draft-edm-protocol-greasing.md

```diff
@@ -156,6 +156,35 @@ cause a new class of interoperability failure. Therefore a label such as
 "reserved for greasing" may help to avoid the confusion.
 
 
+# Deployment Considerations and Incentives for Greasing
+
+Greasing can be used as a tool to improve the active use of existing protocol
+elements (which weren't necessarily designed with greasing to begin with, or
+weren't deployed with greasing); or as part of new protocol design and deployments.
+
+When greasing isn't used from the beginning of protocol deployment, starting to use
+greasing comes with the risk of triggering failures. These failures might be innocuous,
+but they also might be very impactful and visible to users. This risk creates a
+disincentive to deploy greasing in existing systems, since generally the change that
+triggers failures is often blamed for the failure.
+
+Some approaches to avoid failures due to greasing include:
+
+- Designing, implementing, and using greasing very early on in protocol development
+and deployment. This avoids the risks of adding greasing later.
```

Comment:
/risks/ ... are they risk? concern? issues?

Reply:
In the context of the text above it should probably be "risk". If that is the only thing we want to say, a different form could be

".. This avoids the aformentioned late deployment risk."